README.rst

```diff
@@ -40,14 +40,19 @@ Or with `pre-commit <https://pre-commit.com/>`__ in the ``repos`` section of you
         hooks:
         -   id: django-upgrade
             args: [--target-version, "4.1"]   # Replace with Django version
+            types: [file, python]
 
-
-To upgrade your entire project run:
+To upgrade your entire project immediately, you can tell pre-commit to run the ``django-upgrade`` hook on all files. Once you commit these changes, your other pre-commit linters will execute and you may see further changes. Use the following command:
 
 .. code-block:: sh
 
     pre-commit run django-upgrade --all-files
 
+.. tip:: Leaving this hook in place after the upgrade process is considered entirely safe and actually recommended! Django-upgrade may receive future improvements and thus keep improving your code.
+
+.. tip:: Normally, pre-commit will run exactly on the files that you have changed in your commit. Pre-commit can also help you update to the latest ``django-upgrade`` version with its ``autoupdate`` command. This is a sufficient flow for most cases, but remember to also run ``pre-commit run django-upgrade --all-files`` as django-upgrade may add new improvements in each release.
```

Comment:
I think we don't need to explain a lot about how pre-commit works here, just a sentence at most.

Also it's always lower case "pre-commit"